forem/forem
# Place sidebars on tag page in an aside element

## What type of PR is this? (check all applicable)


## Description
Content should be contained within landmark regions to enable users with assistive technology to navigate a page more effectively. 

axe DevTools currently reports that the sidebars on the tags page are not inside a landmark. This pull request replaces the `div` tag used in the sidebars with an `aside` as these sidebars contain complementary content
 
![Tag page with axe tools open. 11 issues reported, one of which is "All page content should be contained by landmarks"](https://user-images.githubusercontent.com/5287128/135611089-12da031d-ff3d-4065-bbb2-23476a53d48d.png)

## Related Tickets & Documents
Resolves #14838 

## QA Instructions, Screenshots, Recordings

- Navigate to a tag page (either via the popular tags list or by clicking one of the tags in an article)
- Run a scan on the page using [axe DevTools](https://www.deque.com/axe/devtools/) and confirm that axe does not report any issues regarding missing landmarks

axe DevTools report after changes:
![Tag page with axe tools open. 10 issues reported](https://user-images.githubusercontent.com/5287128/135611993-bec8c469-0a00-447d-a74d-2aef9d23b74f.png)

### UI accessibility concerns?

Assistive technology such as screen readers can use landmark regions to enable users to navigate easily between different sections on the page (Reference: [All page content must be contained by landmarks Axe Rules | Deque University](https://dequeuniversity.com/rules/axe/4.3/region))

#### How it was tested
- Used axe DevTools as mentioned in QA instructions above
- Used [VoiceOver rotor](https://support.apple.com/en-au/guide/voiceover/mchlp2719/mac) to view what [landmarks were available on the page](https://support.apple.com/en-au/guide/voiceover/vo35709/mac). I couldn't figure out how to screenshot VoiceOver rotor's menu but I did see the two sidebars were listed as `complementary`

## Added/updated tests?

- [x] No, and this is why: I wasn't sure if there were any automated tests in this area that would be affected
